src/Instrumentation/Laravel/tests/Integration/LaravelInstrumentationTest.php

```diff
@@ -73,6 +84,122 @@ public function test_cache_log_db(): void
         $this->assertSame(json_encode(['test' => true]), $logRecord->getAttributes()->toArray()['context']);
     }
 
+    public function test_eloquent_operations(): void
+    {
+        /** @var class-string<\Illuminate\Database\Eloquent\Model> */
+        $modelClass = eval('
+            if (!class_exists("TestModel")) {
+                class TestModel extends \Illuminate\Database\Eloquent\Model
+                {
+                    protected $table = "test_models";
+                    protected $fillable = ["name"];
+                }
+            }
+            return TestModel::class;
+        ');
```

Comment:
I think this should be possible to avoid the eval:

```suggestion
        $model = new class() extends \Illuminate\Database\Eloquent\Model {
            protected $table = 'test_models';
            protected $fillable = ['name'];
        };
```